src/memory.c

```diff
@@ -502,7 +502,7 @@ static int memory_read_internal (value_list_t *vl)
 	}
 
 	/* Unfortunately, the AIX documentation is not very clear on how these
-	 * numbers relate to one another. The only thing is states explcitly
+	 * numbers relate to one another. The only thing is states explicitly
```

Comment:
While you're here fixing this typo, you might as well fix the other one (s/is/it/)